app/components/meetups/mainstage/Mainstage.js

```diff
@@ -0,0 +1,56 @@
+import { useEffect, useState } from 'react';
+import { Collapse } from 'react-bootstrap';
+import styles from '../../../styles/meetup.module.css';
+import dynamic from 'next/dynamic';
+import useWindowSize from '../../hooks/useWindowSize';
+import { ChatToolBar } from './Toolbar';
+import { YoutubeVideoComponent } from './VideoEmbedComponent';
+
+const RCComponent = dynamic(() => import('rc-component-react').then((mod) => mod.RCComponent), {
+  ssr: false,
+});
+
+export const MeetupMainstage = () => {
+  const { width } = useWindowSize()
+
+  const [open, setOpen] = useState();
+  useEffect(() => {
+    if (width < 790 && open === undefined) {
+      setOpen(true)
+    }
+  }, [width, open])
+
+  return (
+    <div className={styles.mainstage_root}>
+      <YoutubeVideoComponent
+        videoId={'E7wJTI-1dvQ'}
```

Comment:
Please pass the video id as one of the props to the `MeetupMainstage` component instead of setting it here. 
Since this will make it easier for the community builder to modify it at the page level instead of looking at the component code.

To summarize, if a community builder has to play a certain video, they will use the component as `<MeetupMainstage videoId={"E7wJTI-1dvQ"} />`

Thank you!

Reply:
Ohh yup, just fixed the parametrization of the component. Thanks a ton @Dnouv!